cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -481,3 +512,67 @@ func isValidChart(chart *models.Chart) (bool, error) {
 	}
 	return true, nil
 }
+
+// GetInstalledPackageSummaries returns the installed packages managed by the 'helm' plugin
+func (s *Server) GetInstalledPackageSummaries(ctx context.Context, request *corev1.GetInstalledPackageSummariesRequest) (*corev1.GetInstalledPackageSummariesResponse, error) {
+	namespace := request.GetContext().GetNamespace()
+	actionConfig, err := s.actionConfigGetter(ctx, namespace)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Unable to create Helm action config: %v", err)
+	}
+	cmd := action.NewList(actionConfig)
+	if namespace == "" {
+		cmd.AllNamespaces = true
+	}
+
+	cmd.Limit = int(request.GetPaginationOptions().GetPageSize())
+	cmd.Offset, err = pageOffsetFromPageToken(request.GetPaginationOptions().GetPageToken())
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, "Unable to intepret page token %q: %v", request.GetPaginationOptions().GetPageToken(), err)
+	}
+
+	// TODO(mnelson): Check whether we need to support ListAll (status == "all" in existing helm support)
+
+	releases, err := cmd.Run()
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Unable to run Helm List action: %v", err)
+	}
+
+	installedPkgSummaries := make([]*corev1.InstalledPackageSummary, len(releases))
+	for i, r := range releases {
+		installedPkgSummaries[i] = installedPkgSummaryFromRelease(r)
+	}
+
+	response := &corev1.GetInstalledPackageSummariesResponse{
+		InstalledPackagesSummaries: installedPkgSummaries,
+	}
+	if len(releases) == cmd.Limit {
+		response.NextPageToken = fmt.Sprintf("%d", cmd.Limit+1)
+	}
+	return response, nil
+}
+
+func installedPkgSummaryFromRelease(r *release.Release) *corev1.InstalledPackageSummary {
```

Comment:
Looking forward to starting using this backend to get rid of the helm2 to helm3 converters we currently have out there :P

Reply:
Also, totally unrelated to this PR, but I've just recalled one issue with TZ we had in the past.
https://github.com/kubeapps/kubeapps/pull/2387/files
Perhaps we should add this envar to the kubeappsapis binary as well at some point.